subsys/usb/class/netusb/function_rndis.c

```diff
@@ -884,7 +867,7 @@ static int handle_encapsulated_cmd(u8_t *data, u32_t len)
 	return 0;
 }
 
-#if SEND_MEDIA_STATUS
+#if IS_ENABLED(SEND_MEDIA_STATUS)
```

Comment:
Could you please add a comment why is it disable per default?

Reply:
I think this should be `#if defined(SEND_MEDIA_STATUS)` instead.

Reply:
> I think this should be `#if defined(SEND_MEDIA_STATUS)` instead.
I have `SEND_MEDIA_STATUS` defined to `0`.

Reply:
> Could you please add a comment why is it disabled per default?

For some reason this option does not work with Windows (8), probably I miss something but it works fine without this media status, probably we do not need it. Maybe I just remove it completely.